apps/laboratory/src/pages/library/wagmi.tsx

```diff
@@ -18,8 +18,12 @@ const wagmiAdapter = new WagmiAdapter({
 
 const modal = createAppKit({
   adapters: [wagmiAdapter],
-  networks: ConstantsUtil.EvmNetworks,
+  networks: wagmiAdapter.caipNetworks,
   projectId: ConstantsUtil.ProjectId,
+  chainImages: {
```

Comment:
We could remove. I've added this to prove Viem chains could be used but not they are available out of the box

Reply:
let's remove 🙏